flocker/node/functional/test_docker.py

```diff
@@ -726,13 +728,15 @@ def test_command_line(self):
         name = random_name(self)
         d = self.start_container(
             name, image_name=u"busybox",
-            command_line=[u"sh", u"-c", u"""\
+            # Pass in pvector since this likely to be what caller actually
+            # passes in:
+            command_line=pvector([u"sh", u"-c", u"""\
```

Comment:
Probably should have a test for at least two types, maybe pvector and list (or a test using an entirely custom sequence implementation ...).